code/modules/mob/living/silicon/robot/life.dm

```diff
@@ -11,6 +11,15 @@
 	if(!stat)
 		use_power()
 
+/mob/living/silicon/robot/handle_environment(datum/gas_mixture/environment)
+	if(!environment)
+		return
+
+	//var/environment_heat_capacity = environment.heat_capacity()
+	var/loc_temp = get_temperature(environment)
+	if(loc_temp > 450)
+		adjustBruteLoss(4)

```

Comment:
Fireloss is miserable to repair on cyborgs. It requires a human opening up the robot, taking the cell out, and mucking with the wires. I thought that'd be too much of a nerf.